dwds/lib/src/debugging/debugger.dart

```diff
@@ -367,7 +367,7 @@ class Debugger extends Domain {
     // TODO(alanknight): Can these be moved to dart_scope.dart?
     final properties = await visibleProperties(debugger: this, frame: frame);
     final boundVariables = await Future.wait(
-      properties.map((property) async => await _boundVariable(property)),
+      properties.map((property) async => _boundVariable(property)),
```

Comment:
```suggestion
      properties.map(_boundVariable)
```

I'd have to measure to be sure, but removing an `async` along with the `await` does have a chance at bringing perf improvements. In the cases where it was the _only_ `await` in the method it could be worth removing along with the `async` - even if for only style reasons.

This case is nice because it can go further and become a tearoff. I'd make this change independent of the rest of the PR.

Reply:
will do, thanks! I wasn't sure if there is any change in semantics:)